lib/fastlane/plugin/android_emulator/actions/disable_android_animations.rb

```diff
@@ -0,0 +1,55 @@
+require 'fastlane/action'
+require_relative '../helper/android_emulator_helper'
+
+module Fastlane
+  module Actions
+    module SharedValues
+    end
+
+    class EnableAndroidAnimationsAction < Action
```

Comment:
1. If you only use the adb-command in an action, you can use this: https://docs.fastlane.tools/actions/adb/
2. Thus the both actions are pretty much the same, I would prefer it as one.
3. I'm not entirely sure, it belongs in this plugin, because it is not emulator-only.

Reply:
On reflection I think you're right that the animation toggle doesn't belong here -- I think it does belong in the screengrab action though, so I might submit a pull request there (and also suggest that it picks up the adb path from the sdk location!)